docker-compose/apim-is-as-km-with-analytics/README.md

```
@@ -5,15 +5,15 @@
  * Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose)
    in order to run the steps provided in following Quick start guide. <br><br>
  * In order to use Docker images with WSO2 updates, you need an active WSO2 subscription.
-   If you don't have a valid WSO2 Subscription, you need to build Docker images by source. Build Docker images using Docker resources available in [here](../../dockerfiles/) and replace the `docker.wso2.com/` prefix from the `image` name in the `docker-compose.yml`. <br><br>
+   If you don't have a valid WSO2 Subscription, you need to build Docker images by source. Build Docker images using Docker resources available in [here](../../dockerfiles/) and replace the `registry.wso2.com/` prefix from the `image` name in the `docker-compose.yml`. <br><br>
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)